client/src/components/answers/TopAnswer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+
+const TopAnswer = props => {
```

Comment:
Spacing nit, I'd recommended doing two spaces for indentation. Ultimately there are no hard requirements but 2 spaces is conventional these days in the js world